### PR TITLE
chore: updating jest to solve npm security issues.

### PR DESCRIPTION
**Pull request by Pruxis**
I updated jest to solve over a 100 security concerns after npm installing.

I had to update the regex because for some reason it didn't seem to work with the previous one.

### CODE REVIEW

**Comment by Pruxis** [comment]
I'll re-install with yarn tonight np

**Comment by Pruxis** [comment]
Oops I installed with my work's NPM registry 🤦‍♂

File: yarn.lock [hunk 1]
```diff
@@ -13,12 +13,32 @@
...
-"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.0.0-beta.35":
+"@babel/code-frame@^7.0.0":
   version "7.0.0"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
   dependencies:
     "@babel/highlight" "^7.0.0"
 
+"@babel/core@^7.1.0":
+  version "7.4.5"
+  resolved "https://npm.tools.chatlayer.ai/@babel%2fcore/-/core-7.4.5.tgz#081f97e8ffca65a9b4b0fdc7e274e703f000c06a"
```
Comment (JoviDeCroock) [inline yarn.lock]: Fancy, your own npm reg

**Comment by Pruxis** [comment]
@JoviDeCroock Comments should be fixed, haha

**Pull request merged**